### DISCUSSION TITLE
Bot attack classname

### DISCUSSION THREAD

**Discussion by shelydiviy** [rehlds/ReGameDLL_CS#1027] [Ideas]
Hello.
There is no possibility to make bots attack certain objects on maps, by classname or in any other way?
Also, is there no possibility to output in ConVar “how well bots earn”? And put some commands on the bot's attack to execute?
It would help to adapt bots and give more realistic behavior on mod servers. Zombie Mod, CSDM like.

**Comment by wopox1337** [comment]
According to the plans there was discussion about adding an API for ReAPI.
These features are not currently available, but, they can be added using the API. You can follow this in the ReGameDLL_CS repositories and then in ReAPI.
